hackathon_site/dashboard/frontend/src/components/dashboard/OverviewCard/OverviewCard.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import Typography from "@material-ui/core/Typography";
+import styles from "./OverviewCard.module.scss";
+import Container from "@material-ui/core/Container";
+import Button from "@material-ui/core/Button";
+import MemoryIcon from "@mui/icons-material/Memory";
+import PeopleIcon from "@mui/icons-material/People";
+import AccountBoxIcon from "@mui/icons-material/AccountBox";
+import LocalMallIcon from "@mui/icons-material/LocalMall";
+import BrokenImageIcon from "@mui/icons-material/BrokenImage";
```

Comment:
I think you are using a different version of MUI Icon, and you should change the version back to what we have. 

The import of the correct version should look like this:
`import LaunchIcon from "@material-ui/icons/Launch";`